### PR TITLE
Feature/feature new database connection

### LINKED ISSUE #103: The permissions message appear in the background
When starting the app, the permissions message appears in the background so it cannot be accessed until the application is sent the background.

### LINKED ISSUE #104: The app closes when it is started for the first time
When you are installing the app for the first time it crashes and you need to restart it

### PR DESCRIPTION
**Pull request by AlbertPG**

### CODE REVIEW

**Review by danicm47 — DISMISSED** [review]

**Review by AlvarTB — DISMISSED** [review]
Good job!

File: app/src/test/java/org/pesmypetcare/mypetcare/services/StubCommunityService.java [hunk 1]
```diff
@@ -55,7 +55,7 @@ private static void addForums() {
         forum.addTag("important");
         new Forum("Washing", "John Doe", DateTime.Builder.buildFullString("2020-04-22T10:00:00"),
             StubCommunityService.groups.get(DINOSAURS));
-        new Forum("Sickling", "John Doe", DateTime.Builder.buildFullString("2020-04-22T10:10:00"),
+        Forum forum2 = new Forum("Sickling", "John Doe", DateTime.Builder.buildFullString("2020-04-22T10:10:00"),
```
Comment (santidrj) [inline app/src/test/java/org/pesmypetcare/mypetcare/services/StubCommunityService.java:58]: Codacy Issue found: [Distance between variable 'forum2' declaration and its first usage is 5, but allowed 4.  Consider making that variable final if you still need to store its value in advance (before method calls that might have side effects on the original value).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5556584)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5556584&bid=18222924)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ app/src/main/java/org/pesmypetcare/mypetcare/controllers/medication/TrObtainAllPetMedications.java  16
         

Coverage decreased per file
===========================
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/user/TrObtainUser.java  -17
- app/src/main/java/org/pesmypetcare/mypetcare/features/community/forums/Forum.java  -17
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/community/TrDeletePost.java  -27
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5556584&bid=18222924)

**Review by AlvarTB — APPROVED** [review]

**Review by xavier-campos-diaz — APPROVED** [review]

**Pull request merged**